### ISSUE TITLE
Case Style did not import

### ISSUE THREAD

**Issue by agboise** [boisebrigade/courtbot#26]
All fields from CSV import as expected except Case Style, which does not appear after import.

**Comment by tylersamples** [comment]
``` 
%Courtbot.Party{
    __meta__: #Ecto.Schema.Metadata<:loaded, "party">,
    case: #Ecto.Association.NotLoaded<association :case is not loaded>,
    case_id: "1482bf02-53f7-4eeb-a44f-b4688d0be29e",
    case_name: "In The Matter of the Application of: Carrie Oakey, For a Change of Name.",
    first_name: nil,
    id: "6442bcbd-6552-49ab-b2c0-6d8523ac0229",
    inserted_at: ~N[2019-04-17 21:37:34],
    last_name: nil,
    updated_at: ~N[2019-04-17 21:37:34]
  }
```
Looks like the CaseStyle was imported but the way I was constructing the "parties" variable in the response had some bad logic and defaulted to the first_name and last_name being used even though they were null.

**Comment by tylersamples** [comment]
Should be fixed in 0.3.0-beta3
